### ISSUE TITLE
Matlab/Julia/Python: Improved data logging functions

### ISSUE THREAD

**Issue by robbr48** [Hopsan/hopsan#2023]
**Is your improvement request related to a problem? Please describe.**
Right now data variables can only be fetched from hopsanc interfaces (Matlab/Julia/Python) through the `getDataVector` function, which works for only one variable. This is not very nice if you want to log all variables from the simulation.

**Describe the solution you'd like**
New functions:
- `getAllDataVectors`
- `disableLogging`
- `enableLogging`

The two latter functions should enable or disable logging for specified variables, preferably using wildcards.

**Comment by OwenDavid** [comment]
While using the python API I have noticed that the variable port names are hard to determine correctly.

For example, if we use the getDataVector() function incorrectly, the error states,

>Component name, port name and variable name must be specified.

If the component name is incorrect then all the components in the model are printed for the user. This is useful.

An improvement would be that in the event that the port is incorrect that all the ports for the component are listed. Same with the variable.

In other words, if the user enters an incorrect component - list all components. If the user enters a correct component but incorrect port, list the ports for the correct component, and if the component and port are correct but the variable is not, then list all the available variables for the given component port.
